Testapp crashes when resuming OffRouteDetectionActivity

App crashes when resuming `OffRouteDetectionActivity` without adding destination.
Entering `OffRouteDetectionActivity` -> Press back button (resume the activity) -> Crash
NPE:
```
3342-3342/com.mapbox.services.android.testapp E/AndroidRuntime: FATAL EXCEPTION: main
                                                                                   Process: com.mapbox.services.android.testapp, PID: 3342
                                                                                   java.lang.RuntimeException: Unable to pause activity {com.mapbox.services.android.testapp/com.mapbox.services.android.testapp.nav.OffRouteDetectionActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.os.Handler.removeCallbacks(java.lang.Runnable)' on a null object reference
                                                                                       at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3381)
                                                                                       at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3340)
                                                                                       at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:3315)
                                                                                       at android.app.ActivityThread.-wrap13(ActivityThread.java)
                                                                                       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1362)
                                                                                       at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                       at android.os.Looper.loop(Looper.java:148)
                                                                                       at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                                                       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                                    Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.os.Handler.removeCallbacks(java.lang.Runnable)' on a null object reference
                                                                                       at com.mapbox.services.android.testapp.nav.OffRouteDetectionActivity.stopSimulation(OffRouteDetectionActivity.java:375)
                                                                                       at com.mapbox.services.android.testapp.nav.OffRouteDetectionActivity.onPause(OffRouteDetectionActivity.java:144)
                                                                                       at android.app.Activity.performPause(Activity.java:6348)
                                                                                       at android.app.Instrumentation.callActivityOnPause(Instrumentation.java:1311)
                                                                                       at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3367)
                                                                                       at android.app.ActivityThread.performPauseActivity(ActivityThread.java:3340) 
                                                                                       at android.app.ActivityThread.handlePauseActivity(ActivityThread.java:3315) 
                                                                                       at android.app.ActivityThread.-wrap13(ActivityThread.java) 
                                                                                       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1362) 
                                                                                       at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                       at android.os.Looper.loop(Looper.java:148) 
                                                                                       at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                                       at java.lang.reflect.Method.invoke(Native Method) 
                                                                                       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

cc/ @zugaldia

--- a/mapbox/app/src/main/java/com/mapbox/services/android/testapp/nav/OffRouteDetectionActivity.java
+++ b/mapbox/app/src/main/java/com/mapbox/services/android/testapp/nav/OffRouteDetectionActivity.java
@@ -372,7 +372,9 @@ public class OffRouteDetectionActivity extends AppCompatActivity {
   } // End startSimulation
 
   private void stopSimulation() {
-    handler.removeCallbacks(runnable);
+    if (handler != null && runnable != null) {
+      handler.removeCallbacks(runnable);
+    }
   }
 
   public static double computeHeading(LatLng from, LatLng to) {

